### PR TITLE
Fixing permission

### PR DESCRIPTION
**Pull request by cloudhandler**

### CODE REVIEW

**Comment by Lauragra** [comment]
@cloudhandler , did you mean to set the Ready for content review label, or are you expecting a different review? Just wanted to check. Thanks.

**Comment by cloudhandler** [comment]
> @cloudhandler , did you mean to set the Ready for content review label, or are you expecting a different review? Just wanted to check. Thanks.

Sorry @Lauragra - yes, meant to use the content review label, my apologies

File: api-reference/beta/api/signin-confirmcompromised.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Allows admins to mark an event in the Azure AD sign in logs as risky. Events marked as risky by an admin are immediately flagged as high risk in Azure AD Identity Protection, overriding previous risk states. Admins can confirm that events flagged as risky by Azure AD Identity Protection are in fact risky or they can mark unflagged events as risky. 
+Allows admins to mark an event in the Azure AD sign in logs as risky. Events marked as risky by an admin are immediately flagged as high risk in Azure AD Identity Protection, overriding previous risk states. Admins can confirm that events flagged as risky by Azure AD Identity Protection are in fact risky and move them to high risk. For more information on investigating Identity Protection risks, visit see [How to investigate risk](/azure/active-directory/identity-protection/howto-identity-protection-investigate-risk).
```
Comment (FaithOmbongi) [inline api-reference/beta/api/signin-confirmcompromised.md:15]:
```suggestion
Allows admins to mark an event in the Azure AD sign in logs as risky. Events marked as risky by an admin are immediately flagged as high risk in Azure AD Identity Protection, overriding previous risk states. Admins can confirm that events flagged as risky by Azure AD Identity Protection are in fact risky and move them to high risk. For more information on investigating Identity Protection risks, see [How to investigate risk](/azure/active-directory/identity-protection/howto-identity-protection-investigate-risk).
```

**Review by FaithOmbongi — APPROVED** [review]
Stumbled onto this PR.

Thank you @cloudhandler for raising this fix. Approved with a suggestion for a minor fix.

**Pull request merged**